src/posix/net.cpp

```diff
@@ -61,12 +61,12 @@ bool SockInfo::createAssociateEvent()
     return true;
 }
 
-bool SockInfo::checkEvent(bool& read, bool& write)
+bool SockInfo::checkEvent(bool& read, bool& write, bool logErr)
 {
     WSANETWORKEVENTS wne;
     memset(&wne, 0, sizeof(wne));
     auto err = WSAEnumNetworkEvents(fd, NULL, &wne);
-    if (err)
+    if (err && logErr)
```

Comment:
I feel this change may have side effects. For safety, I would change it this way:
``` cpp
if (err)
{
         if (logErr)
        {
            auto e = WSAGetLastError(); 
            LOG_err << "WSAEnumNetworkEvents error " << e;
        }
        return false;
}

Reply:
Good point, I'll do that, thanks.   Also, I saw you removed some trailing spaces - I've now installed a Visual Studio plugin so that I can see those :-)  cheers